LongWarOfTheChosen/Src/LW_FactionBalance/Classes/X2DownloadableContentInfo_LW_FactionBalance.uc

```diff
@@ -28,6 +28,8 @@ static event OnPostTemplatesCreated()
 	UpdateDeflect();
 	UpdateShadow();
 	UpdateRemoteStart();
+	//Allows the 2-2-2 faction soldier roster
+	EditReward();

```

Comment:
This doesn't really make it obvious what's being changed. Edit what reward? What's 2-2-2 mean? (Yes, I know, but only because I've seen you reference it in Discord). I would just call the function `AllowTwoSoldiersFromEachFaction()`. Yes it's more wordy, but it's pretty clear. You don't have to go with that specific function name, but please use something that's a bit more informative. (The other function names might benefit from similar treatment, but let's leave those alone for now).